### PR TITLE
IS_MONGODB_CLOUD_URL turn into boolen

### PR DESCRIPTION
**Pull request by tusheer**
In connectWithDb function when we collect IS_MONGODB_CLOUD_URL from env it gives us a value as a string, So if we check is it false or true it always returns true. So I turn it into a boolean using the Boolean function.

### CODE REVIEW

**Comment by foyzulkarim** [comment]
Hi @tusheer bro. Thanks for your pull request. 
However, it seems this fix actually won't fix the problem. :) 
image

**Comment by foyzulkarim** [comment]
@tusheer bro you can try to use `JSON.parse` function to evaluate the variable. 
image

Nice find. I appreciate your efforts

File: server/src/core/mongo.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const mongoose = require("mongoose");
 require('dotenv').config()
-const isMongoDbUrl = process.env.IS_MONGODB_CLOUD_URL;
+const isMongoDbUrl = Boolean(process.env.IS_MONGODB_CLOUD_URL);
```
[the remark this answers is not in the document]
Comment (tusheer) [reply server/src/core/mongo.js:3]: Thank you Bhaiya for your correction. This is really inspiring when I can communicate with you at least through the code. Happy coding Bhaiya.

File: server/src/core/mongo.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 const mongoose = require("mongoose");
 require('dotenv').config()
-const isMongoDbUrl = Boolean(process.env.IS_MONGODB_CLOUD_URL);
+const isMongoDbUrl = JSON.parse(process.env.IS_MONGODB_CLOUD_URL);
```
Comment (foyzulkarim) [inline server/src/core/mongo.js:3]: if `IS_MONGODB_CLOUD_URL` is `undefined` then we will face the same problem. Let's try `JSON.parse(a ? a : 'false')` algo for now. 😎
Thanks 👍

Comment (foyzulkarim) [reply server/src/core/mongo.js:3]: image

**Pull request merged**